package.json

```diff
@@ -186,8 +184,6 @@
 		"@automattic/webpack-extensive-lodash-replacement-plugin": "^0.0.2",
 		"@automattic/webpack-inline-constant-exports-plugin": "^0.0.1",
 		"@automattic/wp-babel-makepot": "^1.0.0",
-		"@automattic/wpcom-block-editor": "^1.0.0-alpha.0",
-		"@automattic/wpcom-editing-toolkit": "^2.7.0",
```

Comment:
All these dependencies are declared in the root `package.json` to make sure that these packages, although not used by the Calypso app, are still part of the `node_modules` tree and their dependencies are properly installed.

At least that's what was needed to make `npm install` work. Is it different with Yarn? Does Yarn install all the subpackages just by virtue of them being declared as workspaces?

There might also be some Lerna workflows that depend on this.

If all this is no longer needed with Yarn, we can remove all the root `package.json` dependencies that point to monorepo packages.

Reply:
> Does Yarn install all the subpackages just by virtue of them being declared as workspaces?

I think that's the case. 

When I did `yarn` in the root after removing this dependencies, the side of `node_modules` was still 1.5Gb, meaning they got installed. Also if the build is not failing to build the apps or run the e2e tests it would mean it can find the dependencies (we only do `yarn install` in the root in CI).

> we can remove all the root package.json dependencies that point to monorepo packages.

Definitely. I'm working on a separate PR to remove all dependencies in the root `package.json` that are not used, monorepo packages will be part of that clean up.
AFAIK the only lerna workflow in use is the one that calls `prepare`

Reply:
> AFAIK the only lerna workflow in use is the one that calls `prepare`

And that workflow loops through the packages as declared in `lerna.json`, and doesn't need the dependencies declared in `package.json`, is that right? That's very nice, means we can remove many lines of spam from `package.json` 🙂